airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/commands.py

```diff
@@ -108,8 +133,19 @@ async def publish(
     python_registry_token: Secret,
     python_registry_url: str,
     python_registry_check_url: str,
+    promote_release_candidate: bool,
+    rollback_release_candidate: bool,
 ) -> bool:
 
+    if promote_release_candidate and rollback_release_candidate:
+        raise click.UsageError("You can't promote and rollback a release candidate at the same time.")
```

Comment:
These should all be mutually exclusive, right?

Reply:
Ah, gaslighting the RC and saying that it definitely is going above and beyond, but no promotion this time. Classic.